.github/workflows/ci.yml

```diff
@@ -25,18 +25,7 @@ jobs:
       - name: Check PHP Version
         run: php -v
 
-      - name: Cache Composer packages
-        id: composer-cache
-        uses: actions/cache@v2
-        with:
-          path: vendor
-          key: ${{ runner.os }}-php-${{ matrix.php }}-composer-${{ hashFiles('**/composer.json') }}-${{ matrix.prefer }}-
-          restore-keys: |
-            ${{ runner.os }}-php-${{ matrix.php }}-composer-${{ matrix.prefer }}-
-
-      - name: Install dependencies
-        if: steps.composer-cache.outputs.cache-hit != 'true'
-        run: composer update --prefer-${{ matrix.prefer }} --prefer-dist --no-progress
+      - uses: "ramsey/composer-install@v1"
```

Comment:
This package handles composer cache out of the box: https://github.com/ramsey/composer-install#about